web/app/themes/mitlib-child/templates/page.php

```diff
@@ -2,63 +2,54 @@
 /**
  * Template Name: Standard Child
  *
- * This is the template that displays all pages by default.
- * Please note that this is the WordPress construct of pages
- * and that other 'pages' on your WordPress site will use a
- * different template.
+ * This page template is an override of the Standard Template which is inherited
+ * from the Parent theme.
+ *
+ * Please note that while this template does load the Main Sidebar defined by
+ * the Child theme, it does _not_ load the "Below Content Widget Area" sidebar.
```

Comment:
Is the `Below Content Widget Area` not conditional-able like the other sidebar? I'm curious if staff can activate something in that area but not have them show up because the template doesn't support it similar to the confusion trying to be eliminated in some of the other work in this PR. If content in that section is different, that's fine but at this point I don't fully understand why we are doing a dedicated template for one type of sidebar and having the other be conditional based on what the content team has activated. (Not asking for a change so much as understanding)

Reply:
That widget area is conditional-able, yes. It looks like I need to update this file comment, because the `content-page.php` partial it calls on line 43 actually _does_ load that sidebar now. I forgot to update this - will push a new commit.